File: laravel-app/app/Models/PollOption.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Concerns\HasUuids;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class PollOption extends Model
{
    /** @use HasFactory<\Database\Factories\PollOptionFactory> */
    use HasFactory, HasUuids;
    protected $table = 'poll_options';

    protected $fillable = [
        'id',
        'poll_id',
        'display_order',
        'option_text'
    ];

    public function poll()
    {
        return $this->belongsTo(Poll::class, 'poll_id');
    }
    public function votes()
    {
        return $this->hasMany(Vote::class, 'option_id');
    }
}

File: netwinform-app/Models/PollOption.cs
using System;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace netwinform_app.Models
{
    public class PollOption
    {
        [JsonProperty("id")]
        public string Id { get; set; }
        [JsonProperty("poll_id")]
        public string PollId { get; set; }
        [JsonProperty("display_order")]
        public int DisplayOrder { get; set; }
        [JsonProperty("option_text")]
        public string OptionText { get; set; }
    }
}
